desktop/core/src/desktop/lib/fs/ozone/__init__.py

```diff
@@ -67,10 +67,15 @@ def _prep(uri):
     except ValueError:
       return '/' if is_root(uri) else uri
   joined = posixpath.join(*list(map(_prep, comp_list)))
-  if joined and joined[0] == '/':
-    joined = 'ofs:/%s' % joined
   return joined
 
+def _serviceid_join(path, ofs_serviceid):
+  if path and (path == '/' or path.startswith('/' + ofs_serviceid)):
+    path = 'ofs:/' + path
+  elif path and not path.startswith(OFS_ROOT + ofs_serviceid + '/'):
```

Comment:
What does `OFS_ROOT` normally look like?

Reply:
`ofs://` 
We will list ozone service id under root

<img width="1435" alt="Screenshot 2023-03-28 at 8 01 45 AM" src="https://user-images.githubusercontent.com/36241930/228112047-a5e87add-aaed-4df8-97c4-c1baf5c67cf8.png">

Reply:
So, is it possible to get a `path` like `ofs://service_id2/xyz`?

Reply:
As a conclusion with ozone team, there will be only 1 service id for per ozone cluster. In the future is it possible.

Reply:
right now if you try to access the path with different id it will show warning

<img width="482" alt="Screenshot 2023-03-28 at 8 13 04 AM" src="https://user-images.githubusercontent.com/36241930/228113703-bd3194b1-5d93-45f2-84fa-4459d3d84f60.png">

Reply:
Sounds good. Let's drop a comment here saying that this function only supports single service-id. So, that whenever we start supporting more, we know that this needs to be updated.

Reply:
As per the Ozone team, there is 1:1 mapping between serviceID and HttpFS role.
```
Always 1 service ID which we need to show in Hue per HTTPfs role - 99.99 percent customer use case is this.

For 2 service IDs means 2 HTTPfs roles - 2 HTTPfs urls so 2 Hue filebrowsers.
```